scenes/waka-waka/Scenes/player.gd

```diff
@@ -1,41 +1,48 @@
 extends CharacterBody2D
 
-# Movement parameters
+## Movement parameters
 @export var max_speed: float = 200.0 # Maximum speed the character can reach
 @export var acceleration: float = 500.0 # How quickly the character speeds up
 @export var deceleration: float = 300.0 # How quickly the character slows down
 
-@onready var sprite: AnimatedSprite2D = $AnimatedSprite2D
+@onready var sprite: AnimatedSprite2D = get_node("PlayerSprite")
 
 func _ready() -> void:
-	velocity = Vector2.ZERO
-	motion_mode = MOTION_MODE_FLOATING
+    velocity = Vector2.ZERO
+    motion_mode = MOTION_MODE_FLOATING
+
+    ## Shader needs some work, but kinda works like this:
+    #var shader_material = ShaderMaterial.new()
+    #shader_material.shader = load("res://scenes/waka-waka/Assets/ghost_shader.gdshader")
+    #sprite.material = shader_material
+
+    sprite.play("idle")
```

Comment:
The character's animation is only playing on movement. My read is that it should play the idle animation when not moving

Reply:
Yeah, I originally had shitty made-by AI ghost animations, and replaced them with this. Ran out of steam/decided to move on after making just the idle animation. :)